helpers/utils.js

```diff
@@ -0,0 +1,45 @@
+const serviceWrapper = (service, fn, env) => {
+    const services = {
+        twilio: phoneNumber => (phoneNumber !== '+17372263943' ? true : new Error('Dev Mode, twilio error')),
+        sendGrid: () => true,
+        actionLimit: () => true,
+        recaptcha: () => true,
+        steem: endpoint => {
+            const endpoints = {
+                'conveyor.is_email_registered': false,
+                'conveyor.is_phone_registered': false,
+                'conveyor.set_user_data': true,
+            };
+            return endpoints[endpoint];
+        },
+    };
+    const fnToCall = env === 'development' ? services[service] : fn;
+    return fnToCall;
+};
+
+const databaseWrapper = (action, fn, env) => {
```

Comment:
Ideally we should test the database logic in debug mode as well

Reply:
Ok. Will do.